.github/workflows/reusable_testing.yml

```diff
@@ -12,6 +12,26 @@ on:
 jobs:
   run_tests:
     runs-on: ubuntu-20.04
+
+    strategy:
```

Comment:
This workflow is mostly for testing of tarantool rather than tarantool-python. I am not sure that there is sense to do testing on different python versions and msgpack library versions here. Such testing makes sense to test the connector itself (like you do in testing.yml). IMHO, for testing of integration of tarantool and the connector the current variant of workflow is enough.

Reply:
Restored reusable_testing